env.JAVA_HOME undefined

I was getting these errors while building the project:

> [ERROR]   The project com.betfair.cougar:cougar-test-utils:3.1-SNAPSHOT (/home/andresp/git/cougar3/cougar-test/cougar-test-utils/pom.xml) has 1 error
> [ERROR]     'dependencies.dependency.systemPath' for com.sun:tools:jar must specify an absolute path but is ${env.JAVA_HOME}/lib/tools.jar @ line 47, column 16
> [ERROR]  
> [ERROR]   The project com.betfair.cougar:cougar-client-code-tests:3.1-SNAPSHOT (/home/andresp/git/cougar3/cougar-test/cougar-client-code-tests/pom.xml) has 1 error
> [ERROR]     'dependencies.dependency.systemPath' for com.sun:tools:jar must specify an absolute path but is ${env.JAVA_HOME}/lib/tools.jar @ line 127, column 25

replacing ${env.JAVA_HOME} with ${java.home} on the referred lines fixed the problem.

--- a/cougar-test/cougar-client-code-tests/pom.xml
+++ b/cougar-test/cougar-client-code-tests/pom.xml
@@ -124,7 +124,7 @@
             <groupId>com.sun</groupId>
             <artifactId>tools</artifactId>
             <scope>system</scope>
-            <systemPath>${env.JAVA_HOME}/lib/tools.jar</systemPath>
+            <systemPath>${java.home}/../lib/tools.jar</systemPath>
             <version>${java.version}</version>
         </dependency>
 

--- a/cougar-test/cougar-test-utils/pom.xml
+++ b/cougar-test/cougar-test-utils/pom.xml
@@ -44,7 +44,7 @@
 			<groupId>com.sun</groupId>
 			<artifactId>tools</artifactId>
 			<scope>system</scope>
-			<systemPath>${env.JAVA_HOME}/lib/tools.jar</systemPath>
+			<systemPath>${java.home}/../lib/tools.jar</systemPath>
             <version>${java.version}</version>
 		</dependency>
 

